### PR TITLE
Metro stop estimate stop number fixed

### PR DESCRIPTION
**Pull request by Divinare**

### CODE REVIEW

File: src/main/java/fi/hsl/transitdata/metroats/MetroEstimatesFactory.java [hunk 1]
```diff
@@ -121,10 +121,17 @@ public MetroEstimatesFactory(final PulsarApplicationContext context) {
...
+        }
+        Integer direction = Integer.parseInt(directionString.get());
+
         // routeRows
         List<MetroAtsProtos.MetroStopEstimate> metroStopEstimates = new ArrayList<>();
         for (MetroStopEstimate metroStopEstimate : metroEstimate.routeRows) {
-            Optional<MetroAtsProtos.MetroStopEstimate> maybeMetroStopEstimate = toMetroStopEstimate(metroStopEstimate);
+            Optional<MetroAtsProtos.MetroStopEstimate> maybeMetroStopEstimate = toMetroStopEstimate(metroStopEstimate, direction);
             if (!maybeMetroStopEstimate.isPresent()) {
                     return Optional.empty();
```
Comment (kenkus-futurice) [inline src/main/java/fi/hsl/transitdata/metroats/MetroEstimatesFactory.java:136]: Should we discard all stop estimates if one fails? Dunno.

**Review by kenkus-futurice — APPROVED** [review]
Looks ok

File: src/main/java/fi/hsl/transitdata/metroats/MetroEstimatesFactory.java (hunk 1 above)
Comment (Divinare) [reply src/main/java/fi/hsl/transitdata/metroats/MetroEstimatesFactory.java:136]: Now we discard all stop estimates for 1 metro. We do the same thing if for example metroEstimate.trainType is missing etc.

I think this is fine, we log it out so if this happens, you can see it from the logs.

Comment (Divinare) [reply src/main/java/fi/hsl/transitdata/metroats/MetroEstimatesFactory.java:136]: Also if one metro estimate row is empty, we discard the whole metro estimate (at line 135)

**Pull request merged**